### PR TITLE
Better explain preselection for items

### PR DESCRIPTION
**Pull request by svdimitr**
I found those as needing additions. 

Fixed the preselect syntax where i found it

### CODE REVIEW

File: components/calendar/selection.md [hunk 1]
```diff
@@ -28,12 +29,15 @@ To control how many dates the user can select, use the `SelectionMode` property.
...
-
-To pre-select dates in the `Multiple` selection mode, use the `SelectedDates` property which is of type `List<DateTime>`.
 
 In `Range` selection mode you can get the start and end dates of the range the user selected through the `RangeStart` and `RangeEnd` parameters of type `DateTime`. You also get events `RangeStartChanged` and `RangeEndChanged`. You can read more about them and see an example in the [Events]({%slug components/calendar/events%}) article.
 
+## Preselection of Dates
+
+You can preselect a date:
+* In `Single` selection mode by setting the `Value` property of the calendar to the desired date.
+* In `Multiple` selection mode, use the `SelectedDates` property which is of type `List<DateTime>`.
```
Comment (marin-bratanov) [inline components/calendar/selection.md:39]: what about range selection?

**Comment by svdimitr** [comment]
Marked this as Draft Pull Request since i am continuing my work on it.

File: components/calendar/selection.md [hunk 2]
```diff
@@ -38,6 +38,56 @@ You can preselect a date:
 * In `Single` selection mode by setting the `Value` property of the calendar to the desired date.
 * In `Multiple` selection mode, use the `SelectedDates` property which is of type `List<DateTime>`.
 
+>caption Preselect date / dates in the Calendar
+
+````CSHTML
```
Comment (marin-bratanov) [inline components/calendar/selection.md:44]: I recommend you split this into two separate examples

**Comment by svdimitr** [comment]
I have put thought into this and how it might best be presented. What do you think of a preselection section in all Selection Overview articles? @marin-bratanov

**Comment by marin-bratanov** [comment]
Sure, it makes sense to separate it out in a dedicated example.

**Pull request closed without merging**